tests/robottelo/test_datafactory.py

```diff
@@ -47,7 +47,7 @@ def test_filtered_datapoint_True(self):
                         return_value=False):
```

Comment:
Not needed, but does not make any harm either and can be used in case other datafactory function would be altered by bz :)
But I think I'll remove that to keep code cleaner.